UnityProject/Assets/Scripts/Items/Tool/Pickaxe.cs

```diff
@@ -18,6 +18,11 @@ public class Pickaxe : MonoBehaviour, ICheckedInteractable<PositionalHandApply>,
 		[SerializeField]
 		private float timeMultiplier = 1f;
 
+		[Range(1, 10)]
+		[Tooltip("what degree of hardness this mining tool can overcome. Higher means the pick can mine more types of things.")]
+		[SerializeField]
+		private int pickHardness = 5;
```

Comment:
would you like this to be an arbitrary integer or would it be better to use a finite amount of hardness levels to define it?

Reply:
isn't a limit range integer a finite amount of hardness levels?

Reply:
yeah, I guess 10 levels might be too many levels. I don't know, wouldn't a drop down where you pick between say 4 levels be easier to manage?

Reply:
I don't think four values is enough for the effect I'm trying to achieve. I mean, we're dealing with 5 tiers of pickaxe, 2 drills, the jackhammer, PKA, PKC, 2 versions of plasma cutter, and the mech drill. The goal of this hardness scale is to try to (along with later additions) create a dynamic where you get utility out of all kinds of mining tools because they have different strengths and weaknesses. 

I originally drafted around 5 points of hardness, but I felt like it was too limiting for how many tools we have. As an example, with a 5 point scale, you'd naturally end up with a pickaxe in each tier (improvised, compact, standard, silver-tipped, diamon-tipped), and then the drills and jackhammer being 3, 4, and 5, because 1 and 2 are too low for high tier tools created by R&D. Thing is, this defeats the point of the PR as the drills would be the same hardness as the picks in their tier, and the point is to try to make drills _not automatically better_ than picks in all situations, like they currently are.

tl;dr, I thought about it, and I think the scale needs to be higher than 4-5 values. I went with 10 because it provides enough granularity to do the job and as a bonus, the real mohs scale of hardness is based on a 1 to 10 scale.